server/src/main/java/org/apache/druid/client/cache/HybridCache.java

```diff
@@ -131,6 +133,14 @@ public void close(String namespace)
     level2.close(namespace);
   }
 
+  @Override
+  @LifecycleStop
+  public void close() throws IOException
+  {
+    level1.close();
```

Comment:
Better to organise this in a double empty try-with-resources idiom:
```java
try (Closeable ignore = level1; Closeable ignore2 = level2) {
  // Just for closing
}
```
To delegate boilerplate exception and suppression handling to the Java construct.

If it appears unnatural, you could extract this in a utility method, e. g. called
```
void closeBoth(Closeable, Closeable) throws IOException
```